ompi/mca/pml/ucx/pml_ucx.h

```diff
@@ -57,6 +57,7 @@ struct mca_pml_ucx_module {
     mca_pml_ucx_freelist_t    convs;
 
     int                       priority;
+    int                       cuda_initialized;
```

Comment:
where is it set to 0?

Reply:
@yosefe initialized to "false" in mca_pml_ucx_open() (https://github.com/open-mpi/ompi/pull/7893/files#diff-80c6bd864dd92a9d6cdcfa297313cc9cR247)